packages/formspree-core/src/utils.ts

```diff
@@ -24,100 +22,14 @@ export const clientHeader = (givenLabel: string | undefined): string => {
   return `${givenLabel} ${label}`;
 };
 
-export const appendExtraData = (
+export function appendExtraData(
   formData: SubmissionData,
   prop: string,
   value: string
-) => {
+): void {
   if (formData instanceof FormData) {
     formData.append(prop, value);
   } else {
-    formData = Object.assign(formData, { [prop]: value });
-  }
-};
-
-type HandleSCAargs = {
-  stripePromise: Stripe;
-  response: Response;
-  responseData: any;
-  payload: {
-    paymentMethod: PaymentMethod;
-    error?: undefined;
-  };
-  data: SubmissionData;
-  fetchImpl: (
-    input: RequestInfo,
-    init?: RequestInit | undefined
-  ) => Promise<Response>;
-  request: {
-    method: string;
-    mode: 'cors';
-    body: string | FormData;
-    headers: {
-      [key: string]: string;
-    };
-  };
-  url: string;
-};
-
-export const handleSCA = async ({
-  stripePromise,
-  response,
-  responseData,
-  payload,
-  data,
-  fetchImpl,
-  request,
-  url,
-}: HandleSCAargs): Promise<SubmissionResponse> => {
```

Comment:
This is now inlined in `submitForm`, since it's the only caller.

It's more confusing as a separate function as we need to construct the mental map between the actual value passed from `submitForm` and the arguments here.